tests/Data/dataSource.cpp

```diff
@@ -75,7 +75,7 @@ void _testDataSource( const std::string& uriStr )
 BOOST_AUTO_TEST_CASE( memoryDataSource )
 {
     std::stringstream volumeName;
-    volumeName << "mem:///#" << VOXEL_SIZE_X << "," << VOXEL_SIZE_Y << ","
+    volumeName << "mem://#" << VOXEL_SIZE_X << "," << VOXEL_SIZE_Y << ","
```

Comment:
"men:///#" is valid , but even "mem:#whatever" is a valid URI. The // separator is needed when you are going to specify an authority, to distinguish it from an absolute path (a single /).